### PR TITLE
Use the Google UUID library

### PR DESCRIPTION
**Pull request by rycus86**
Replaces the Satori UUID library that has a vulnerability: Insecure Randomness problem https://snyk.io/vuln/SNYK-GOLANG-GITHUBCOMSATORIGOUUID-72488 (https://cwe.mitre.org/data/definitions/338.html)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,8 +16,8 @@ vet:
 .PHONY: lint
 lint:
 	@echo "============= Go lint ==============="
-	@go get github.com/golang/lint/golint
-	@test -z "$$(go list ./... | xargs -L1 "$(GOPATH)"/bin/golint | tee /dev/stderr)"
+	@go get golang.org/x/lint/golint
+	@test -z "$$(go list ./... | xargs -L1 "$(GOROOT)"/bin/golint | tee /dev/stderr)"
```
Comment (rycus86) [inline Makefile]: I had to do this change on my machine but would be great if someone could try whether this is working or I'm doing it wrong here.

Comment (justincormack) [reply Makefile]: This seems to be working for me (if you clone under `GOPATH` anyway). Can you revert for now, as I think its correct as is...

Comment (rycus86) [reply Makefile]: Sure. Just revert the `GOROOT` to `GOPATH` right? The package location should change to `golang.org/x/lint/golint` if I'm reading the error message right?

Comment (justincormack) [reply Makefile]: yes, golang.org/x/lint/golint is the correct path. This should probably be in a different commit as it is unrelated.

Comment (rycus86) [reply Makefile]: Tried it in a `golang:1.12` container, it was fine under `GOPATH`. Thanks!

**Comment by rycus86** [comment]
Force pushed the change with the `Makefile` changed in its own commit.

**Comment by thaJeztah** [comment]
ping @mason-fish PTAL

**Pull request merged**